wca-regulations.md

```diff
@@ -417,7 +417,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - A7b1) If penalties are assigned, the judge records the original recorded time displayed on the timer, along with any penalties. The format should be "X + T + Y = F", where X represents the sum of time penalties before/starting the solve, T represents the time displayed on the timer (the "original recorded time"), Y represents a sum of time penalties during/after the solve, and F represents the final result (e.g. 2 + 17.65 + 2 = 21.65). If X and/or Y is 0, the 0 terms are omitted (e.g. 17.65 + 2 = 19.65).
     - A7c) The judge and competitor must each check the recorded result, and sign (or initial) the score sheet to acknowledge that the result is correct, complete, properly formatted, and clearly readable. This finishes the attempt.
         - A7c1) If the competitor or the judge refuses to accept and sign the score sheet, the WCA Delegate must resolve the dispute.
-        - A7c2) The competitor must not sign an attempt on the score sheet before the judge has recorded the final result for that attempt. If a competitor signs (or otherwise marks) an attempt before they have begun it, or before the judge has finished recording the result, they forfeit the attempt and the result will be considered DNS.
+        - A7c2) The competitor must not sign an attempt on the score sheet before the judge has recorded the final result for that attempt. If a competitor signs (or otherwise marks) an attempt before they have begun it, they forfeit the attempt and the result will be considered DNS. If the competitor signs (or otherwise marks) an attempt before the judge has finished recording the result, it will be considered DNF.
```

Comment:
I think this is not quite sufficient. Technically, "If the competitor signs (or otherwise marks) an attempt before the judge has finished recording the result, it will be considered DNF." also applies before the attempt has started.

I think the best way to make this clear would be to split it into `A7c2` (signing before the attempt) and `A7c3` (signing after the start of the attempt, but before the judge has written the result.